cartridges/docker/apache-hbase-1.0.1.1/run.sh

```diff
@@ -34,7 +34,7 @@ startWkaMember() {
 
 startMember() {
 	name="hbase-${memberId}"
-	container_id=`docker run -e CONFIG_PARAM_HDFS_HOST=localhost -e CONFIG_PARAM_ZOOKEEPER_HOST=localhost -e CONFIG_PARAM_HBASE_MASTER="${wka_member_ip}" -e CONFIG_PARAM_HBASE_MASTER_HOSTNAME="${wka_member_container_id}" -d wso2/hbase:1.0.1.1`
+	container_id=`docker run -e CONFIG_PARAM_HDFS_HOST=172.17.0.30 -e CONFIG_PARAM_ZOOKEEPER_HOST=172.17.0.34 -e CONFIG_PARAM_HBASE_MASTER="${wka_member_ip}" -e CONFIG_PARAM_HBASE_MASTER_HOSTNAME="${wka_member_container_id}" -d wso2/hbase:1.0.1.1`
```

Comment:
Hbase need both Hadoop and Zookeeper ip addresses before hand. This run.sh will need only when running docker containers without ppaas.